Scroller/EditableWrapper.js

```diff
@@ -260,6 +271,36 @@ const EditableWrapper = (props) => {
 		}
 	}, [centered]); // TODO: Need dataSize dependency for centeredOffset
 
+	useEffect(() => {
+		// addEventListener to moveItems while scrolled
+		const container = scrollContentRef.current;
+		const wrapperWidth = wrapperRef.current.getBoundingClientRect().width;
+		const scrollContentCenter = container.getBoundingClientRect().width / 2;
+
+		const handleMoveItemsByScroll = () => {
```

Comment:
I expected call handleMouseMove simply.
I guess it is because animation.
I think We need to discussion that remove animation for selected Item.


```suggestion
		
		const handleMoveItemsByScroll = () => {
			const {centeredOffset, itemWidth, mouseClientX, fromIndex, selectedItem} = mutableRef.current;

			if(selectedItem) {
				selectedItem.classList.add(css.stopAnimation); // should remove when finish scroll
			}
			handleMouseMove({clientX: mouseClientX});
		}
```